lib/__tests__/watch.test.ts

```diff
@@ -0,0 +1,191 @@
+import chokidar from 'chokidar';
+import PQueue from 'p-queue';
+
+import { uploadFolder } from '../cms/uploadFolder';
+import { logger } from '../logger';
+import { watch } from '../cms/watch';
+import { MODE } from '../../constants/files';
+
+jest.mock('chokidar');
+jest.mock('axios');
+jest.mock('p-queue');
+jest.mock('../cms/uploadFolder');
+
+describe('watch function', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
TypeScript was being overly strict and requiring that I mock out chokidar in its entirety. Opted for this solution instead.